snare/cloner.py

```diff
@@ -14,6 +14,8 @@
 
 class Cloner(object):
     def __init__(self, root, max_depth, css_validate):
+        self.logger = logging.getLogger(__name__)
+        self.logger.setLevel(logging.DEBUG)
```

Comment:
This doesn't work (at least for me). Have you run it?

The handler of the parent logger, which created in cloner bin file has a level ERROR, guess this is the problem